CONTRIBUTING.md

```diff
@@ -108,6 +108,11 @@ sense to place a `const` directly below the user (especially if there is a
 single user, or just a few co-located users).
 
 The `#[cfg(test)] mod tests {}` module goes on the very bottom, if present.
+Other module definitions (like `mod foo { .. }`) can be ordered among other
+items as it makes sense in the context of the items imported from them.
+Module declarations (like `mod foo;`) should be ordered before other items
+but after imports. Imports from local modules (both declared and defined)
+should be kept close to the module declaration/definition.
```

Comment:
This loses the advice from #2264  about not putting many `mod` definitions alongside non-trivial code. Is that intentional?

Reply:
SGTM. TY